src/commands/folder/spec/folder.spec.server.ts

```diff
@@ -0,0 +1,191 @@
+import dotenv from 'dotenv'
+import { folder } from '../index'
+import { generateTimestamp } from '../../../utils/utils'
+import { ServerType, Target } from '@sasjs/utils/types'
+import {
+  removeFromGlobalConfig,
+  saveToGlobalConfig
+} from '../../../utils/config'
+import { Command } from '../../../utils/command'
+
+describe('sasjs job execute', () => {
+  let target: Target
+
+  beforeAll(async (done) => {
+    target = await createGlobalTarget()
+    process.projectDir = process.cwd()
+
+    done()
+  })
+
+  afterAll(async (done) => {
+    await removeFromGlobalConfig(target.name)
+    done()
+  })
+
+  it('list folder children', async (done) => {
+    const timestamp = generateTimestamp()
+    const testFolderPath = `/Public/app/cli-tests/cli-tests-folder-${timestamp}`
+
+    await createTestFolder(testFolderPath, target.name)
+
+    const commandList1 = new Command(
+      `folder list /Public/app/cli-tests -t ${target.name}`
+    )
+    const list1 = await folder(commandList1)
+    expect(list1).toContain(`cli-tests-folder-${timestamp}`)
+
+    await deleteTestFolder(testFolderPath, target.name)
+    done()
+  })
+
+  it('move folders keeping the folder name', async (done) => {
+    const timestamp = generateTimestamp()
+    const testFolderPath = `/Public/app/cli-tests/cli-tests-folder-${timestamp}`
+
+    await createTestFolder(testFolderPath, target.name)
+
+    const commandList1 = new Command(
+      `folder list /Public/app/cli-tests -t ${target.name}`
+    )
+    const list1 = await folder(commandList1)
+    expect(list1).toContain(`cli-tests-folder-${timestamp}`)
+
+    const commandCreate2 = new Command(
+      `folder create ${testFolderPath}/temp/test -t ${target.name}`
+    )
+    await folder(commandCreate2)
+
+    const commandCreate3 = new Command(
+      `folder create ${testFolderPath}/test2 -t ${target.name}`
+    )
+    await folder(commandCreate3)
+
+    const commandMove1 = new Command(
+      `folder move ${testFolderPath}/temp ${testFolderPath}/test2 -t ${target.name}`
+    )
+    await folder(commandMove1)
+
+    // Check if operations are executed correctly
+    let folderList1 = await folder(
+      new Command(`folder list ${testFolderPath}/test2 -t ${target.name}`)
+    )
+    expect(folderList1).toContain('temp')
+
+    let folderList2 = await folder(
+      new Command(`folder list ${testFolderPath}/test2/temp -t ${target.name}`)
+    )
+    expect(folderList2).toContain('test')
+
+    await deleteTestFolder(testFolderPath, target.name)
+    done()
+  })
+
+  it('move folder to same location renaming the folder', async (done) => {
```

Comment:
`move folder to same location renaming the folder` -> `move folder to the same location and rename it`